src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -24,6 +24,9 @@ namespace Microsoft.CodeAnalysis
     /// </summary>
     internal sealed class PEModule : IDisposable
     {
+        // We need to store reference for to keep the metadata alive while symbols have reference to PEModule.
+        private readonly ModuleMetadata _owner;
```

Comment:
Why do we need this now, how did we manage without it so far and what has changed?